docs/api/specs/webConnectionProtocol.md

```diff
@@ -0,0 +1,384 @@
+---
+id: webConnectionProtocol
+sidebar_label: Web Connection Protocol
+title: Web Connection Protocol (next)
+---
+
+:::info _[@experimental](../fdc3-compliance#experimental-features)_
+
+FDC3's Web Connection Protocol (WCP) is an experimental feature added to FDC3 in 2.2. Limited aspects of its  design may change in future versions and it is exempted from the FDC3 Standard's normal versioning and deprecation polices in order to facilitate any necessary change.
+
+:::
+
+The FDC3 Web Connection Protocol (WCP) defines the procedure for a web-application to connect to an FDC3 Desktop Agent. The WCP is used to implement a [`getAgent()`](../ref/GetAgent) function in the [`@finos/fdc3` npm module](https://www.npmjs.com/package/@finos/fdc3), which is the recommended way for web applications to connect to a Desktop Agent. This specification details how it retrieves and provides the FDC3 `DesktopAgent` interface object and requirements that Desktop Agents must implement in order to support discovery and connection via `getAgent()`. Please see the [`getAgent` reference document](../ref/GetAgent.md) for its TypeScript definition and related types.
```

Comment:
"must"
Almost all occurrences of the word "must" on this page are capitalized as "MUST".  However, if you search you can see there are (I think) 5 occurrences that are just in lower case.  Unless this is a deliberate choice, these occurrences should be updated (I won't say _must_ be updated!)

Reply:
Fixed in 5e8ba8b01ab5ecf86081172ff87369b1faf28540